src/Runner.js

```diff
@@ -99,7 +99,7 @@ function getAllFiles(paths, filter) {
     paths.map(file => new Promise((resolve, reject) => {
       fs.lstat(file, (err, stat) => {
         if (err) {
-          process.stdout.error('Skipping path ' + file + ' which does not exist. \n');
```

Comment:
That's the result of a bad merge from my side. I didn't replace those instances properly :-/ Thanks for fixing that!